### PR TITLE
docs: add toolkit docs

### PR DESCRIPTION
**Pull request by ShyunnY**
This PR mainly supplements the documentation for new toolkit features.

Mainly focuses on the following three points:
+ trace: Supplemented `AddEvent` documentation

+ logging: Added `toolkit/logging` usage documentation

+ metric: Added `toolkit/metric` usage documentation

### CODE REVIEW

**Comment by wu-sheng** [comment]
You should update menu.yml to keep them visible on website.

File: docs/en/advanced-features/manual-apis/toolkit-metric.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+### MeterOpt
+
+MeterOpt is a common Option for metric types. Currently, only `WithLabels` is supported to attach labels to the metric.
+
+```go
+// WithLabels Add labels for metric
+func WithLabels(key, val string) MeterOpt
+```
```
Comment (wu-sheng) [inline docs/en/advanced-features/manual-apis/toolkit-metric.md:94]: At the end of metric docs and logs docs, you should link to latest backend documents for metrics and logs analysis.

Users may not know, they need to write MAL or customize log process through LAL.

Comment (ShyunnY) [reply docs/en/advanced-features/manual-apis/toolkit-metric.md:94]: Sorry, I may not understand: `link to latest backend documents for metrics and logs analysis`.

I don't quite understand where the link goes? Can you give an example? Thanks a lot :)

Comment (wu-sheng) [reply docs/en/advanced-features/manual-apis/toolkit-metric.md:94]: I mean

- link to here from metrics toolkit doc https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-meter/
- https://skywalking.apache.org/docs/main/latest/en/setup/backend/log-agent-native/ need to update to add go agent
- link to here from log toolkit doc https://skywalking.apache.org/docs/main/latest/en/setup/backend/log-analyzer/

Comment (ShyunnY) [reply docs/en/advanced-features/manual-apis/toolkit-metric.md]: Done. Meanwhile, should I update the description of Log-Agent in the main document to add go-agent?

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
Please submit a pull request to the mail repo for the log reporter.

**Review by mrproliu — APPROVED** [review]

**Comment by ShyunnY** [comment]
> Please submit a pull request to the mail repo for the log reporter.

A small question: In what period will the skywalking-go documents be synchronized to the website? I need to reference this in log reporter for the main website repo

**Pull request merged**